### PR TITLE
fix incorrect setting of max_new_tokens in warmup

### PR DESCRIPTION
**Pull request by sywangyi**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by sywangyi** [comment]
@regisss @kdamaszk please help review

**Review by regisss — APPROVED** [review]
LGTM, waiting for @kdamaszk's green light

File: router/client/src/client.rs [hunk 1]
```diff
@@ -175,7 +175,7 @@ impl Client {
                     watermark: false,
                 }),
                 stopping_parameters: Some(StoppingCriteriaParameters {
-                    max_new_tokens: 10,
+                    max_new_tokens: std::cmp::min(10,max_total_tokens-max_input_length),
```
Comment (jkaniecki) [inline router/client/src/client.rs]: Please, add spaces, no need to call std::cmp as it's defined in file's head (line 8).
Should be "cmp::min(10, max_total_tokens - max_input_length)"

**Review by jkaniecki — CHANGES_REQUESTED** [review]
LGTM, only some code organisation suggestions

**Review by jkaniecki — APPROVED** [review]

**Pull request merged**